Oware/src/InController.java

```diff
@@ -0,0 +1,70 @@
+import java.util.InputMismatchException;
+import java.util.Scanner;
+
+class InController
+{
+	public int choosePit(Player player, Pit[] pits) throws Exception // Catching a input other than a number between 1 - 12
+	{
+		Scanner input = new Scanner(System.in);
+		int choice = 0;
+		
+		if (player.getWhichPlayer()) // For player choosing from pits 1 - 6
```

Comment:
If the player knew its own store house, pits and opponents pits, the player could harvest its own pits, I think a more robust Player object would make a lot of this easier, but this is a clever minimalist solution.